src/test/java/Tests/RatesOnboarding.java

```diff
@@ -0,0 +1,42 @@
+package Tests;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import PageObject.*;
+
+    public class RatesOnboarding extends Start {
+
+    @Test
+    public void ownerConfiguresRatePlan () throws Exception {
+      loginAndGoToRates();
+
+        //Add Standard and Seasonal pricings
+        OnboardingRates onboardingRates = new OnboardingRates(driver);
+        onboardingRates.ratePricings ();
```

Comment:
It is hard to understand what is going on here. maybe rename it into something more human readable